### ISSUE TITLE
"back to search" links; breadcrumbs

### ISSUE THREAD

**Issue by jrochkind** [sciencehistory/scihist_digicoll#268]
in chf_sufia, there are "back to search" links on "show" pages. (At least most of em). 

This is provided via Blacklight feature to keep track of current search context, with a pretty complicated implementation. It also ends up integrated with sufia "breadcrumbs" feature. 

Previously, it was decided that this *was* an important feature for end-users. But I'm not sure if that's the result of user-testing or just a hunch. 

It's a bit tricky to get working, but we can if needed.

**Comment by jrochkind** [comment]
@HKativa There may have been other relevant "breadcrumbs" in chf_sufia -- I'm not actually sure what may have been there, or what ones we care about. 

Can you confirm that for now, for the purpose of this ticket, it's only "back to search" that we care about?

**Comment by jrochkind** [comment]
Actual blacklight has not only a "back to search" on an item detail page, but also a 'next' / 'previous' within search:

![Screenshot 2020-01-23 15 17 18](https://user-images.githubusercontent.com/149304/73020416-91c25200-3df3-11ea-8390-0b204d876abc.png)

My memory is that even in chf_sufia we had already dispensed with that, it was only "back to search". 

Also, per question above, I am operating on the assumption that there aren't any other "breadcrumbs" we care about other than "back to search" -- there may not have been any others in chf_sufia, I'm not certain, none come to mind.

**Comment by eddierubeiz** [comment]
Yeah this is familiar territory. See e.g. https://github.com/sciencehistory/chf-sufia/pull/1034/files

**Comment by jrochkind** [comment]
Note that the back to search functionality from Blacklight can get very confused if you have more than one tab/window open, and/or access direct links to records. It can take you to the wrong "back to search". 

This was true in chf_sufia too, as well as stock blacklight.

**Comment by jrochkind** [comment]
Also can't work on a Collection page -- if you get a collection in search results, and click on it, you don't get "back to search" on collection page. This was true of chf_sufia too.

**Comment by eddierubeiz** [comment]
I think the Collection case is covered in a separate issue.

**Comment by eddierubeiz** [comment]
Well at any rate see https://github.com/sciencehistory/chf-sufia/issues/885 re: collections.

**Comment by eddierubeiz** [comment]
Never mind the last two comments! I just wanted to make sure that the old somewhat-related Sufia PR and issue were mentioned in connection with Blacklight and back-to-search breadcrumbs. Now I see I already did that a week ago!

**Comment by jrochkind** [comment]
This is almost working, although it would need more documentation as to what's going on, and possibly more tests. 

One thing that's not currently working is if you click on a Collection in search results, you get an error: `RuntimeError in CollectionShow#index: Unable to find track_collection_show_path route helper. Did you add `concerns :searchable` routing mixin to your `config/routes.rb`?
`

(and also not tested, no tests are failing!) 

While we could get it working -- we've actually decided to shelve this for now. We decided it doesn't provide enough user benefit for the cost: it doesn't get a huge amount of clicks on analytics; we haven't gotten any direct feedback from it; users can just use the browser back button.
